test/TesterInternal/StatsTests.cs

```diff
@@ -57,7 +59,8 @@ public StatsInitTests(ITestOutputHelper output, Fixture fixture)
             var clientStatisticsManager = this.HostedCluster.ServiceProvider.GetService<ClientStatisticsManager>();
             Assert.NotNull(clientStatisticsManager); // Client Statistics Manager is setup
 
-            Assert.Equal("MockStats",  config.StatisticsProviderName);  // "Client.StatisticsProviderName"
+            var statisticsOptions = this.HostedCluster.ServiceProvider.GetService<IOptions<StatisticsOptions>>();
```

Comment:
nit: whenever you get IOptions from the container, call .Value before assigning it.
No big deal here, but for consistency... we should almost never explicitly assign a variable to IOptions, it's only a tool for resolving via DI.